django_project/minisass_authentication/admin.py

```diff
@@ -34,11 +34,20 @@ class UserProfileInline(admin.StackedInline):
 
 
 def correct_country(modeladmin, request, queryset):
+    country_mapping = {
```

Comment:
I think it's better to create a new model called CountryMapping and add a fixture for the initial data.